docker/node/server.js

```diff
@@ -197,6 +197,75 @@ LinkedList.Circular.prototype.toArray = function () {
 	return out;
 };
 
+
+LinkedList.Circular.prototype.some = function(cb) {
+	return this.find(cb) !== null;
+}
+
+LinkedList.Circular.prototype.find = function(cb) {
+	let video = this.first;
+
+	for (let i = 0; i < this.length; i++) {
+		if (cb(video, i)) {
+			return video;
+		}
+
+		video = video.next;
+	}
+
+	return null;
+}
+LinkedList.Circular.prototype.multiple = function(indexes) {
+	let map = new Map(
+		indexes.map(i => [i, null])
+	);
+
+	let video = this.first;
+	let max = Math.max(...indexes);
+
+	for (let i = 0; i <= max; i++) {
+		if (map.has(i)) {
+			map.set(i, video);
+		}
+
+		video = video.next;
+	}
+
+	return indexes.map(i => map.get(i));
+}
+LinkedList.Circular.prototype.at = function(index) {
+	let video = this.first;
+
+	for (let i = 0; i < index; i++) {
+		video = video.next;
+	}
+
+	return video;
+}
+
+LinkedList.Circular.prototype.each = function(cb) {
+	let video = this.first;
+
+	for (let i = 0; i < this.length; i++) {
+		cb(video, i);
+		video = video.next;
+	}
+}
+
+LinkedList.Circular.prototype.indexOf = function(cb) {
```

Comment:
nit: I'm guessing this should be `findIndex` if you're wanting to keep aligned with the same naming as the array functions